app/notifiers/course/assessment_notifier.rb

```
@@ -5,4 +5,16 @@ def assessment_attempted(user, assessment)
       notify(assessment.tab.category.course, :feed).
       save
   end
+
+  # To be called when user submitted an assessment.
+  def assessment_submitted(user, course_user, submission)
+    # TODO: Replace with a group_manager method in course_user
+    group_managers = course_user.groups.flat_map { |g| g.group_users.manager }.map(&:course_user)
+    # Default to course manager if the course user do not have any group manager
+    managers = group_managers.count > 0 ? group_managers : course_user.course.managers
+
+    activity = create_activity(actor: user, object: submission, event: :submitted)
+    managers.each { |manager| activity.notify(manager.user, :email) }
+    activity.save
```

Use `save!`, so it won't fail silently.